content/en/contribute/medic/team.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Team Organization"
+linkTitle: "Team Organization"
+weight: 1
+description: >
+   Product Team Organization at Medic
+---
+
+
+### Product Team Leadership 
+|Title|Role|
+|--|--|
+|Chief Product Officer|Leads & aligns the team to achieve ambitious organizational goals|
+|Senior Director of Technology|Ensures that digital health systems and internal processes are built on cohesive set of technologies that can grow in a sustainable manner|
+|Director of Product Development|Directs the processes and teammates to build and maintain the Community Health Toolkit|
```

Comment:
The leadership section takes too much importance, and makes me think we could drop this whole page unless we use it to add much more on the rest of the team. As it is it doesn't really have enough context to be useful for understanding the rest of the sections in the handbook.